src/sentry/models/file.py

```diff
@@ -431,6 +431,12 @@ def assemble_from_file_blob_ids(self, file_blob_ids, checksum, commit=True):
         tf.seek(0)
         return tf
 
+    def delete(self, *args, **kwargs):
+        blob_ids = [blob.id for blob in self.blobs.all()]
```

Comment:
What's the largest number of blobs we're likely to see here in practice?

Reply:
I'll have to find out. This table is so big that we can't query it through redash.

Reply:
Only concern here is if there are a very large number of jobs it could delay this delete, but maybe it's fine either way.